RELEASING.rst

```diff
@@ -1,47 +1,63 @@
+#. Update ``CHANGES.rst`` with a human-readable list of changes since the last commit.
+   ``git log --oneline --no-decorate --color 0.1^..develop`` can be used to list the changes since the last version.
+#. ``git add docs/api CHANGES.rst`` to stage the updated documentation.
+#. ``git commit -m "REL: 0.2"`` to commit the changes.
+#. ``git push origin develop``
+#. ``git checkout master``
+#. ``git merge develop`` (do not create a merge commit)
+#. ``git push origin master``
+#. **Verify that all continuous integration test and build workflows pass.**
+#. Create a release on GitHub
 
-   ``git tag 0.2 master -m "Version 0.2"`` Replace ``0.2`` with the new version.
+   #. Go to https://github.com/pycalphad/pycalphad/releases/new
+   #. Set the "Tag version" field to ``0.2``.
+   #. Set the branch target to ``master``.
+   #. Set the "Release title" to ``pycalphad 0.2``.
+   #. Leave the description box blank.
+   #. If this version is a pre-release, check the "This is a pre-release" box.
+   #. Click "Publish release".
+#. The new version will be available on PyPI when the ``Build and deploy to PyPI`` workflow on GitHub Actions finishes successfully.
 
-   ``git show 0.2`` to ensure the correct commit was tagged.
+Deploy to PyPI (manually)
+-------------------------
 
-   ``git push origin master --tags``
+.. warning::
 
-   ``git stash pop``
-11. The new version is tagged in the repository. Now the public package must be built and distributed.
+   DO NOT FOLLOW THESE STEPS unless the GitHub Actions deployment workflow is broken.
+   Creating a GitHub release should trigger the ``Build and deploy to PyPI`` workflow on GitHub Actions that will upload source and platform-dependent wheel distributions automatically.
 
-Uploading to PyPI
------------------
-1. ``rm -R dist/*`` on Linux/OSX or ``del dist/*`` on Windows
-2. With the commit checked out which was tagged with the new version:
-   ``python setup.py sdist``
+To release a source distribution to PyPI:
 
-   **Make sure that the script correctly detected the new version exactly and not a dirty / revised state of the repo.**
+#. If deploying for the first time: ``pip install twine build``
+#. ``rm -R dist/*`` on Linux/OSX or ``del dist/*`` on Windows
+#. ``git checkout master`` to checkout the latest version
+#. ``git pull``
+#. ``git log`` to verify the repository state matches the newly created tag
 
-   Assuming a correctly configured .pypirc:
+#. ``python -m build --sdist``
+#. **Make sure that the script correctly detected the new version exactly and not a dirty / revised state of the repo.**
+#. ``twine upload dist/*`` to upload (assumes a `correctly configured <https://packaging.python.org/specifications/pypirc/>`_ ``~/.pypirc`` file)
 
-   ``twine upload -r pypi -u rotis dist/*``
 
-Uploading to conda-forge
-------------------------
+Deploy to conda-forge
```

Comment:
Will this manual step still be necessary for conda-forge? Will auto PRs be opened for our PyPI source distributions?

Reply:
PRs will continue to open automatically. This section is out of date and can probably be moved to a similar "manual" heading as PyPI.

I was honestly just too tired of writing all the main releasing and PyPI steps to have the energy to get to this one.